Testing Your Seed.md

```diff
@@ -0,0 +1,34 @@
+# Testing Your Seed
+
+Once you have made the backup of a Seed, this Seed can be used to access your Skywallet. 
+
+**NOTE - The Seed generated for a Skywallet can only be used with a Skywallet. You cannot use this Seed in the Desktop Wallet.**
+
+The **Confirm Seed** button in the Hardware Wallet window of your Desktop Wallet can help you in such situations to verify whether a Seed belongs to a particular Skywallet.
```

Comment:
Please change  this sentence  to: `If you want to check, whether a Seed you have written down belongs to a given Skywallet, you can use the **Confirm Seed** button. This is a  security procedure to check whether the backup has been successful`